pkg/tsdb/testdatasource/scenarios.go

```diff
@@ -283,6 +283,17 @@ func init() {
 		},
 	})
 
+	registerScenario(&Scenario{
```

Comment:
would be nice with a proper backend error scenario as well (like a go panic) so that the http status is 500. Been missing that.

Reply:
added -- and made the behavior in angular vs react the same ;)